hack/release-stable-minor.sh

```diff
@@ -52,4 +61,24 @@ done
 unset GITHUB_TOKEN
 unset WEBHOOK
 DIR="$(dirname "${0}")"
-exec "${DIR}/stabilization-changes.py"
+I=1
+STABILIZATION_LOG="$(mktemp)"
+touch "${STABILIZATION_LOG}"
+
+# Execute stabilization-changes.py until it stops making changes
+while true; do
+	echo "Running stabilization-changes.py: iteration ${I}" | tee -a "${STABILIZATION_LOG}"
+	if ! "${DIR}"/stabilization-changes.py &>> "${STABILIZATION_LOG}"; then
+			echo "FAIL: stabilization-changes.py output tail: (see full log in ${STABILIZATION_LOG})"
```

Comment:
nit: 3 tab indent when the parent line was only 1.  I'd expect two tabs here.

Reply:
that's what I get for using a home computer w/o properly set up tools :D